.github/workflows/cancel_workflows.yml

```diff
@@ -0,0 +1,14 @@
+name: Cancel Workflows
+on:
+  workflow_run:
+    workflows: ["Go", "Golicense", "Kind", "Build and push latest image if needed", "Antrea upgrade"]
+    types:
+      - requested
```

Comment:
I'm reading the webpage you shared: https://github.com/styfle/cancel-workflow-action#advanced-pull-requests-from-forks. I see the following:

> Therefore, a special setup using workflow_run, which also works for push, is needed

Can you make sure that workflows _triggered by push events to the main branch_ are never cancelled. In other words, let's only cancel workflows which are triggered by pull request events. In case of regression, it is useful to have the full workflow "history" for the main branch to detect which merge caused the regression. I'd rather keep pushing the Docker image to the registry for every push to main as well.

I think something like this would work:
```
jobs:
  cancel:
    name: Cancel workflows
    runs-on: ubuntu-latest
    if: ${{ github.event.workflow_run.event == 'pull_request' }}
    steps:
    - uses: styfle/cancel-workflow-action@0.9.0
      with:
        workflow_id: ${{ github.event.workflow.id }}
```

Reply:
sure, after verify a similar job in a sample repo, it will indeed also cancel the workflow triggered by merge, I will check how to make it to work on pull_request only.

Reply:
@antoninbas  I just verified the code you provided is correct, which can help to skip cancel workflows in main stream. and those actions triggered by forks PR can be cancelled as well, and the push on fork branch, I guess I misunderstood it, looks like all the workflows are triggered by PR from fork repo only in antrea repo, so no need to worry about the push on fork branch. I just updated the code. please check. thanks.